apps/web/src/index.css

```
@@ -58,6 +58,23 @@
   }
 }
 
+/* Safe-area inset utilities for iOS devices with a Dynamic Island / notch /
+   home indicator. Used on the topmost chrome surfaces so they stay clear of
+   the camera and gesture areas when the page is rendered with
+   viewport-fit=cover. Insets resolve to 0 on devices without safe areas. */
+@utility pt-safe {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)